cli/wallet/nep17.go

```diff
@@ -57,8 +57,9 @@ var (
 		walletPathFlag,
 		walletConfigFlag,
 		&flags.AddressFlag{
-			Name:  "token",
-			Usage: "Token contract address or hash in LE",
+			Name:     "token",
+			Usage:    "Token contract address or hash in LE",
+			Required: true,
```

Comment:
There's a set of `.IsSet` usages in the code that return error in case if the flag is not set. Search for these usages, where possible mark the appropriate flags as required and remove these checks.

Reply:
Also, there is a question about fromAddrFlag.
In the `multitransfer` and `transfer` commands in usage --from marked as required. But in `func getDefaultAddress(fromFlag *flags.Address, w *wallet.Wallet)` the default one will be used if fromAddrFlag is not provided. So fromAddrFlag is not required? 

But maybe to be more user-friendly is better to make it required. It can be tricky to find out that I need to pass --address when I'm getting such an error: `can't fetch matching token from RPC-node: NEP-17 token was not found`